packages/web/src/components/modal-radio/ModalRadioItem.tsx

```diff
@@ -10,24 +10,29 @@ type ModalRadioItemProps = {
   onClick: (e: any) => void
   disabled?: boolean
   className?: string
+  contentClassName?: string
 }
 
 export const ModalRadioItem = ({
   selected,
   onClick,
   children,
   disabled = false,
-  className
+  className,
+  contentClassName
 }: ModalRadioItemProps) => {
   return (
-    <div
-      className={cn(
-        styles.radioItem,
-        { [styles.selected]: selected, [styles.disabled]: disabled },
-        className
-      )}
-    >
-      <div className={styles.radioItemContent} onClick={onClick}>
+    <div className={cn(styles.radioItem, className)}>
```

Comment:
I still think we should make use of labels/radio inputs and the radio group component, if you get a free cycle.